.github/CODEOWNERS

```diff
@@ -4,78 +4,78 @@
-/plugins/git-release-manager                        @backstage/reviewers @erikengervall
-/plugins/home                                       @backstage/reviewers @backstage/techdocs-core
-/plugins/ilert                                      @backstage/reviewers @yacut
-/plugins/jenkins                                    @backstage/reviewers @timja
-/plugins/jenkins-backend                            @backstage/reviewers @timja
-/plugins/kafka                                      @backstage/reviewers @nirga @andrewthauer
-/plugins/kafka-backend                              @backstage/reviewers @nirga @andrewthauer
-/plugins/kubernetes                                 @backstage/reviewers @backstage/warpspeed
-/plugins/kubernetes-*                               @backstage/reviewers @backstage/warpspeed
-/plugins/newrelic-dashboard                         @backstage/reviewers @mufaddal7
-/plugins/playlist                                   @backstage/reviewers @kuangp
-/plugins/playlist-*                                 @backstage/reviewers @kuangp
-/plugins/rollbar                                    @backstage/reviewers @andrewthauer
-/plugins/rollbar-backend                            @backstage/reviewers @andrewthauer
-/plugins/scaffolder-backend-module-rails            @backstage/reviewers @angeliski
-/plugins/scaffolder-backend-module-yeoman           @backstage/reviewers @pawelmitka
-/plugins/search                                     @backstage/reviewers @backstage/techdocs-core
-/plugins/search-*                                   @backstage/reviewers @backstage/techdocs-core
-/plugins/sonarqube                                  @backstage/reviewers @backstage/sda-se-reviewers
-/plugins/stack-overflow                             @backstage/reviewers @backstage/techdocs-core
-/plugins/stack-overflow-backend                     @backstage/reviewers @backstage/techdocs-core
-/plugins/techdocs                                   @backstage/reviewers @backstage/techdocs-core
-/plugins/techdocs-*                                 @backstage/reviewers @backstage/techdocs-core
-/plugins/user-settings-backend                      @backstage/reviewers @backstage/sda-se-reviewers
-/tech-insights-backend                              @backstage/reviewers @xantier @iain-b
-/tech-insights-backend-module-jsonfc                @backstage/reviewers @xantier @iain-b
-/tech-insights-tech-insights-common                 @backstage/reviewers @xantier @iain-b
-/tech-insights-tech-insights-node                   @backstage/reviewers @xantier @iain-b
+*                                                   @backstage/maintainers
+yarn.lock                                           @backstage/maintainers @backstage-service
+*/yarn.lock                                         @backstage/maintainers @backstage-service
+/.changeset/cost-insights-*                         @backstage/maintainers @backstage/silver-lining
```

Comment:
I think we can remove this one now

Reply:
Also removed us from the `cost-insights/*` too. Maybe that makes sense - will see.